### PR TITLE
Examples for JS driver

### PR DESCRIPTION
**Pull request by michielmulders**
I created some examples based on the docs and added them to the js-driver.
I have code for websocket as well, but the stream doesn't seem to be working.
Gonna do some more tests on this, if it works, I'll add the code for websockets as well.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**Async/Await Basic Usage**: Basic usage example rewritten with async/await.
+-> `npm run basic-async`
+
+**Querying for Assets**: Query for assetdata or metadata.
+-> `npm run query-assets`
+
+**Seed/Keypair Functionality**: Create keypair with bip39 library.
+-> `npm run seed-func`
+
+## Notes
```
Comment (manolodewiner) [inline examples/README.md]: I would rather include this Notes part at the beginning, as otherwise the  Different Examples do not work.

**Comment by michielmulders** [comment]
@manolodewiner fixed the note to the top.

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Quick Notes
+`dotenv` is listed as a dependencies in `package.json`.
+If you want to use this, add a `.env` file to the root of this project (same level as this `README.md` file)
+and replace the variables to fit your specific config.
+
+```
+BIGCHAINDB_API_PATH=http://localhost:9984/api/v1/
+BIGCHAINDB_APP_ID=<your-app-id>
+BIGCHAINDB_APP_KEY=<your-app-key>
```
Comment (manolodewiner) [inline examples/README.md]: If you are running BigchainDB in you own environment i.e. localhost, then app id and app key are not needed and should be null. Maybe is better if you use instead the reference to BigchainDB testnetwok
```js
(
    "https://test.bigchaindb.com/api/v1/",
    {
        app_id: "Get one from testnet.bigchaindb.com",
        app_key: "Get one from testnet.bigchaindb.com"
    }
)
```

Comment (manolodewiner) [reply examples/README.md]: What do you think @michielmulders ?

Comment (jernejpregelj) [reply examples/README.md]: +1 on this

**Pull request merged**